test/unit-and-integration/stake.test.ts

```diff
@@ -1,15 +1,18 @@
 import { cleanup, fireEvent, render, screen } from '@testing-library/svelte';
```

Comment:
Ideas for more tests:
- displays onclick
- closes on cancel
- stake amount is correct
- Requirements are displayed